src/pattern-library/components/patterns/transition/SliderPage.tsx

```diff
@@ -7,16 +7,17 @@ import Library from '../../Library';
 
 const Slider_: FunctionComponent<
   ComponentProps<TransitionComponent> & { _transitionStatus?: 'in' | 'out' }
-> = ({ children, visible, onTransitionEnd, _transitionStatus }) => {
-  const [isVisible, setIsVisible] = useState(visible);
-  const toggleSlider = () => setIsVisible(prev => !prev);
+> = ({ children, direction, onTransitionEnd, _transitionStatus }) => {
```

Comment:
This typing is a little hard to read (the multi-line wrapping). Is it possible to construct the type elsewhere?

Reply:
Actually, this is now redundant, and `_transitionStatus` can be completely discarded in favor of `direction`. It was needed when `visible` was used, but not anymore.

This can be simplified as `const Slider_: TransitionComponent = ...`.

Reply:
Ok no, my mistake. Doing that has other side effects.

I will extract the type in multiple definitions for better readability.